lms/templates/head-extra.html

```diff
@@ -22,6 +22,15 @@
 % if apple_app_id:
   <meta name="apple-itunes-app" content="app-id=${get_global_settings()['apple_app_id']}">
 % endif
+% if get_global_settings().get('custom_site_meta_tags'):
+  % for meta_tag in get_global_settings().get('custom_site_meta_tags'):
+    % if meta_tag['tagName']:
```

Comment:
```suggestion
    % if meta_tag.get('tagName'):
```

Better for failure handling.